docker/config/local_dev.env

```diff
@@ -23,10 +29,11 @@ resource.postgresql.database_port=5432
 secrets.postgresql.database_username=postgres
 secrets.postgresql.database_password=aPassword
 
-# statsd
-# ------
+# metrics
+# -------
 
 resource.statsd.statsd_host=statsd
+resource.statsd.markus_backends=markus.backends.logging.LoggingMetrics,markus.backends.statsd.StatsdMetrics
```

Comment:
I did this as "resource.statsd.markus_backends", but wasn't sure whether to do that or just change all the "resource.statsd" to "resource.metrics".

The latter makes more sense. It probably requires a minor change on the ops side.

Thoughts?

Reply:
I created a PR in the infra repo so they have it before I make the change.

https://github.com/mozilla-services/cloudops-deployment/pull/1836